spec/acceptance/04_plugins_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper_acceptance'
+
+describe 'sensu::plugins class', unless: RSpec.configuration.sensu_cluster do
+  agent = hosts_as('sensu_agent')[0]
+  backend = hosts_as('sensu_backend')[0]
+  context 'on agent' do
+    it 'should work without errors' do
```

Comment:
The `it` text should be updated to show that we're checking idempotency.

`it should work without errors and be idempotent` or you can wrap that idempotency check like this

```ruby
it 'should be idempotent' do
  apply_manifest_on(agent, pp, :catch_changes  => true)
end
```

Reply:
Used the `it should work without errors and be idempotent`.